src/components/LandingPage/LandingPage.jsx

```diff
@@ -18,22 +19,34 @@ function LandingPage() {
                         media="(min-width:481px)"
                         srcSet={logo}
                     />
-                    <img src={logo} alt="poké queen logo" />
-                    {/*//den här ska jag byta ut mot en png-bild för gamla browsers*/}
+                    <img src={logoFallback} alt="poké queen logo" />
                 </picture>
                 <h2>Om du vill vara trendig - och äta gott!</h2>
             </header>
             <section className={styles.promotionText}>
                 <h1>Grand Opening 23/12</h1>
-                <button className="primary-btn">boka nu</button>
+                <button className="primary-btn">Boka nu</button>
             </section>
             <div className={styles.arrowContainer}>
-            <img
-                src={arrow}
-                className={styles.arrow}
-                alt="arrow pointing down"
-            />
+                <img
+                    src={arrow}
+                    className={styles.arrow}
+                    alt="arrow pointing down"
```

Comment:
The alt text should probably be in Swedish of the website's main language is Swedish?